hawkular-inventory-api/src/main/java/org/hawkular/inventory/api/Inventory.java

```diff
@@ -101,6 +110,91 @@
  * @since 0.0.1
  */
 public interface Inventory extends AutoCloseable, Tenants.Container<Tenants.ReadWrite> {
+    static final class InventotyUtils {
+        private static final Map<String, Class<? extends Entity<?, ?>>>entityTypes;
+        static {
+            Map<String, Class<? extends Entity<?, ?>>> m = Arrays.asList(
+                    Tenant.class,
+                    Environment.class,
+                    Feed.class,
+                    ResourceType.class,
+                    MetricType.class,
+                    Resource.class,
+                    Metric.class,
+                    OperationType.class,
+                    MetadataPack.class)
+            .stream().collect(Collectors.toMap(Class::getName, Function.identity()));
+            entityTypes = Collections.unmodifiableMap(m);
+        }
+        static String shortenIfNeeded(CanonicalPath path) {
```

Comment:
This has no place in the API.. This method was introduced to integrate with hwk accounts that didn't like our possibly long paths that are the only thing uniquely identifying an entity. Inventory always stores the full path and I don't think we need to introduce yet another unique identifier for an entity in inventory itself (i.e. having 2 ways of identifying an entity).